lua/hop/init.lua

```diff
@@ -514,6 +514,26 @@ function M.hint_lines(opts)
   )
 end
 
+function M.hint_lines_position(opts)
+  opts = override_opts(opts)
+  -- only makes sense as end position given movement goal.
+  opts.hint_position = require'hop.hint'.HintPosition.END 
+
+  local generator
+  if opts.current_line_only then
+    generator = jump_target.jump_targets_for_current_line
+  else
+    generator = jump_target.jump_targets_by_scanning_lines
+  end
```

Comment:
Does it really make sense to jump to the current line? :)

Reply:
Whoops, that's just boilerplate left over that I copied from the other hint functions. I can change that to just be `local generator = jump_target.jump_targets_by_scanning_lines`, but that would still include the current line. Is there any easy way to exclude the current line, or will we have to implement a new jump_target function?

Reply:
Hm, I think it could make sense to add something to `jump_targets_by_scanning_lines` that exclude the current line to prevent that kind of things, yeah, but probably in another PR.

Now that I think about it, that pattern is present in `hint_lines` as well. Let’s merge your work as-is and I will fix that kind of things (it’s not really important anyway).